gulpfile.js

```diff
@@ -79,6 +80,40 @@ const ensureContainerRemoved = async ({ containerName }) => {
   }
 };
 
+async function ensureBucketExists() {
+  const region = 'eu-central-1';
+  const bucketName = 'dev-educandu-cdn';
+
+  const minioClient = new MinioClient({
+    endPoint: 'localhost',
+    port: 9000,
+    useSSL: false,
+    region,
+    accessKey: MINIO_ACCESS_KEY,
+    secretKey: MINIO_SECRET_KEY
+  });
+
+  const bucketPolicy = {
+    Version: '2012-10-17',
```

Comment:
Quite a new version we're having here :)

Reply:
I think it's the latest on AWS ;-)